interactive_ai/supported_models/geti_supported_models/manifests/detection/deim_l.yaml

```
@@ -0,0 +1,19 @@
+id: Object_Detection_Deim_DFine_l
+name: Deim-DFine-l
```

```suggestion
name: Deim-DFine-L
```
Making it uppercase for consistency with YOLO models (e.g. YOLOX-L)